recipe/meta.yaml

```diff
@@ -26,8 +26,11 @@ requirements:
     - xarray
     - cartopy
     - panel >=0.5.1
+    - holoviews==1.12.1
```

Comment:
Should be `holoviews >=1.12.1`

Reply:
But the latest version of `holoviews` has problem with correctly displaying hovers for lat/lon. So would it be okay?

Reply:
I see. Then you might want a more complicated selector to allow for future releases where that problem will be fixed.
In any case, include the space, and check whether you need one "=" or two.